search/search-client/src/main/java/io/camunda/search/clients/aggregation/SearchAggregation.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.clients.aggregation;
+
+import static io.camunda.search.clients.aggregation.SearchAggregationBuilders.aggregation;
+
+import io.camunda.search.clients.query.SearchQuery.Builder;
+import io.camunda.util.ObjectBuilder;
+import java.util.function.Function;
+
+public record SearchAggregation(SearchAggregationOption aggregationOption) {
```

Comment:
Why do we have this class and in addition `SearchAggregationOption`? Can we merge them into one? It seems that this class's transformer only passes through to the aggregation option class and adds otherwise no value.